### PR TITLE
Restructuring project

### PR DESCRIPTION
**Pull request by JakeChampion**

### CODE REVIEW

File: tests/server/lib/search.spec.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
-'use strict';
+'use strict'; // eslint-disable-line strict
 /* global describe, it, before, beforeEach, afterEach, after */
 const chai = require('chai');
 const chaiAsPromised = require('chai-as-promised');
-const fs = require('fs');
-const path = require('path');
-const searchFixturePath = path.join(__dirname, '..', 'fixtures', 'search.json');
+// const fs = require('fs'); // TODO: Create test that uses fixture data
```
Comment (AdaRoseEdwards) [inline tests/server/lib/search.spec.js]: Delete

File: tests/server/lib/topStories.spec.js [hunk 2]
```diff
@@ -1,10 +1,10 @@
-'use strict';
+'use strict'; // eslint-disable-line strict
 /* global describe, it, before, beforeEach, afterEach, after */
 const chai = require('chai');
```
Comment (AdaRoseEdwards) [inline tests/server/lib/topStories.spec.js]: whitespace, around /**/ comments

File: tests/server/lib/update.spec.js [hunk 3]
```diff
@@ -1,9 +1,9 @@
-'use strict';
+'use strict'; // eslint-disable-line strict
```
Comment (AdaRoseEdwards) [inline tests/server/lib/update.spec.js]: newline

File: client/src/js/svgloader.js [hunk 4]
```diff
@@ -1,3 +1,4 @@
+/* global mina, Snap */
```
Comment (AdaRoseCannon) [inline client/src/js/svgloader.js:1]: Move to vendor directory?

**Pull request merged**